sdcm/nemesis.py

```diff
@@ -5069,6 +5069,11 @@ def disrupt_create_index(self):
         Create index on a random column (regular or static) of a table with the most number of partitions and wait until it gets build.
         Then verify it can be used in a query. Finally, drop the index.
         """
+        with self.cluster.cql_connection_patient(self.target_node) as session:
+            if is_tablets_feature_enabled(session=session):
```

Comment:
I don’t think the issue is specific only for tablets enabled scenario.
You can search for “CreateIndex” in issues and see we have many of them.

Reply:
@roydahan, I suggest merging this and checking the results next week. if it fails - we will update this line.